### PR TITLE
StyledIcon prop on Atom Checkbox

### PR DESCRIPTION
**Pull request by AitorRodriguez990**
## Description

Add a new prop to style the Atom Checkbox.

Related task: https://jira.scmspain.com/browse/SUIC-178

## Preview

image

image

### Preview with label

image

image

### CODE REVIEW

File: demo/molecule/checkboxField/demo/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@s-ui/hoc": "1"
+    "@s-ui/hoc": "1",
+    "@s-ui/react-atom-icon": "1"
```
Comment (midudev) [inline demo/molecule/checkboxField/demo/package.json]: Not needed anymore as we're importing the local file.

**Review by midudev — APPROVED** [review]
Looks good. Thanks for taking the time to get this to the next level. 👏

File: components/atom/checkbox/src/index.js [hunk 2]
```diff
@@ -68,7 +72,10 @@ AtomCheckbox.propTypes = {
   onChange: PropTypes.func,
 
   /* Will set a red/green border if set to true/false */
-  errorState: PropTypes.bool
+  errorState: PropTypes.bool,
+
+  /* If you with you can change the look of checkbox using an AtomIcon */
```
Comment (JoanClaret) [inline components/atom/checkbox/src/index.js]: `If you with you` looks weird, i guess this is a unwanted mistake 🙄

Comment (AitorRodriguez990) [reply components/atom/checkbox/src/index.js]: Done! 😅

**Review by JoanClaret — APPROVED** [review]

File: components/atom/checkbox/src/index.js (hunk 2 above)
Comment (midudev) [reply components/atom/checkbox/src/index.js]: With or withooooou youuu

**Pull request merged**